guides/advanced_guides/snapshots_and_dumps.md

```diff
@@ -0,0 +1,101 @@
+# Snapshots and Dumps
+
+## Snapshot vs Dump
+
+MeiliSearch has two ways to save its data, `snapshots` and `dumps`.
+
+Snapshots make it possible to schedule the creation of hard copies of your database. This feature is **intended mainly as a safeguard**—ensuring that if some failure occurs, you're able to relaunch MeiliSearch quickly and efficiently without having to wait for the re-indexing of documents. They are **not compatible between versions**.
+
+Dumps, at the difference, are **exporting the data out of MeiliSearch** in a way that they are not bound to the MeiliSearch version. Because of that, when importing a dump, MeiliSearch will need to index all of your documents, a process that takes up time and overhead (based on the amount of documents, their size and the settings).
+
+**To summarize**, **snapshots are highly efficient but not portable between different versions of MeiliSearch. Dumps, on the other hand, are highly portable but not very efficient, as frequently launching MeiliSearch from a dump would cause your performance to suffer.
```

Comment:
```suggestion
**To summarize**, snapshots are highly efficient but not portable between different versions of MeiliSearch. Dumps, on the other hand, are highly portable but not very efficient, as frequently launching MeiliSearch from a dump would cause your performance to suffer.
```